domenetjenester/iay/src/main/java/no/nav/foreldrepenger/abakus/domene/iay/arbeidsforhold/ArbeidsforholdInformasjonEntitet.java

```diff
@@ -267,11 +269,16 @@ public EksternArbeidsforholdRef finnEksternRaw(Arbeidsgiver arbeidsgiver, Intern
         if (internReferanse.getReferanse() == null) return EksternArbeidsforholdRef.nullRef();
 
         return referanser.stream()
+            .filter(this::erIkkeMerget)
```

Comment:
samme som for fpsak - burde ikke denne metoden være helt rå?

Reply:
jo